### PR TITLE
Bump minimal requirements

### PR DESCRIPTION
**Pull request by jderusse**
| Q             | A
| ------------- | ---
| Branch?       | 4.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | 
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Follow up #33079 Fix failures with lowest requirements in branch 4.3

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php [hunk 1]
```diff
@@ -1288,8 +1288,8 @@ public function testNoClassFromGlobalNamespaceClassIdWithLeadingSlash()
 
     public function testNoClassFromNamespaceClassIdWithLeadingSlash()
     {
-        $this->expectException('Symfony\Component\DependencyInjection\Exception\RuntimeException');
-        $this->expectExceptionMessage('The definition for "\Symfony\Component\DependencyInjection\Tests\FooClass" has no class attribute, and appears to reference a class or interface. Please specify the class attribute explicitly or remove the leading backslash by renaming the service to "Symfony\Component\DependencyInjection\Tests\FooClass" to get rid of this error.');
+        $this->expectException('Symfony\Component\DependencyInjection\Exception\InvalidArgumentException');
+        $this->expectExceptionMessage('Service definition "\Symfony\Component\DependencyInjection\Tests\FooClass" has no class, and its name looks like a FQCN but it starts with a backslash; remove the leading backslash.');
```
Comment (jderusse) [inline src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php]: Looks like related to #33092. ping @fabpot

Comment (nicolas-grekas) [reply src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php]: From `CheckDefinitionValidityPass`!

**Review by xabbuh — APPROVED** [review]

File: src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php (hunk 1 above)
Comment (ro0NL) [reply src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php]: perhaps im onto something - https://github.com/symfony/symfony/pull/33092#issuecomment-520138148

File: src/Symfony/Component/VarExporter/Tests/VarExporterTest.php [hunk 2]
```diff
@@ -64,11 +64,18 @@ public function provideFailingSerialization()
         };
 
         yield [$a];
+    }
 
-        $a = [null, $h];
-        $a[0] = &$a;
+    /**
+     * This test is identical to testFailingSerialization, but the tested data causes a segfault in PHP 7.2
+     * https://github.com/sebastianbergmann/phpunit/issues/3790.
```
Comment (jderusse) [inline src/Symfony/Component/VarExporter/Tests/VarExporterTest.php]: Opened an issue in PHP https://bugs.php.net/78397

Comment (jderusse) [reply src/Symfony/Component/VarExporter/Tests/VarExporterTest.php]: fixed by #33110

**Comment by nicolas-grekas** [comment]
Thank you @jderusse.

**Pull request merged**